docs/endpoints/campaigns.md

```diff
@@ -11,8 +9,9 @@ GET /api/v3/campaigns
 ### Optional Query Parameters
 
 - **filter[column]** _(string)_
-  - Filter results by the given column: `id`, `is_open`
+  - Filter results by the given column: `id`, `is_open`, `has_cause`
   - You can filter by more than one value for the ID column, e.g. `/campaigns?filter[id]=121,122`
+  - The `has_cause` filter (`/campaigns?filter[has_cause]=education`) will return campaigns _including_ the specified cause. (Can filter by one value only).
```

Comment:
Out of scope for this PR, but I'm curious if we've talked at all about converting the `cause` single value text field into a table like `campaign_causes`?  This is how the Ashes taxonomy worked -- and would allow us to filter by multiple causes, and also avoid using regex to find results...

Reply:
We didn't really have that discussion pertaining to this work, but I agree that that would be a helpful update, and actually formalize cause association! (Definitely feels more relevant now that we're starting to actually _use_ this taxonomy)